lib/entity.js

```diff
@@ -65,6 +66,16 @@ var RESERVED_PROPERTY_NAMES = [
   'remove'
 ];
 
+/** Timeout for azure table requests */
+var AZURE_TABLE_TIMEOUT     = 30 * 1000;
+
+/** Azure table agent used for all instances of the table client */
+var globalAzureTableAgent = new https.Agent({
+  keepAlive:      true,
+  maxSockets:     5000,
+  maxFreeSockets: 1000
```

Comment:
Looks right though these numbers seem insanely high... I think you would be better off with some lower numbers?